### PR TITLE
[Docs](merge-on-write) Add more docs for partial update using native insert statement

### PR DESCRIPTION
**Pull request by bobhan1**
## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhannngchen** [comment]
run buildall

File: docs/zh-CN/docs/advanced/variables.md [hunk 1]
```diff
@@ -682,7 +682,7 @@ try (Connection conn = DriverManager.getConnection("jdbc:mysql://127.0.0.1:9030/
 * `enable_unique_key_partial_update`
 
   <version since="2.0.2">
-  是否在对insert into语句启用部分列更新的语义，默认为 false
+  是否在对insert into语句启用部分列更新的语义，默认为 false。需要注意的是，控制insert语句是否开启严格模式的会话变量`enable_insert_strict`的默认值为true，即insert语句默认开启严格模式，而在严格模式下进行部分列更新时如果插入的数据中存在key列在原表中不存在的行数据，则整个导入会失败。所以，在使用insert语句进行部分列更新的时候如果要允许插入key列在原表中不存在的行数据，需要在`enable_unique_key_partial_update`设置为true的基础上同时将`enable_insert_strict`也设置为true。
```
Comment (zhannngchen) [inline docs/zh-CN/docs/advanced/variables.md:685]: 而在严格模式下进行部分列更新不允许更新不存在的key。所以如果希望进行列更新时能插入不存在的key，需要。。。

**Review by zhannngchen — APPROVED** [review]
LGTM

**Comment by bobhan1** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**